xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.xml

```diff
@@ -126,7 +126,7 @@
 
 === 2.0 Groups ===
 
-Groups can be used to insert another document inline directly into the current document. This allows for example to insert complex elements inside a list item or inside a table cell. Groups are delimited by the following syntactic elements: ##{{{(((...)))}}}##. One Group can contain another Group and there is no limit of imbrication.
+Groups can be used to insert another document or div section inline directly into the current document. This allows for example to insert complex elements or style inside a list item, inside a table cell or a paragraph. Groups are delimited by the following syntactic elements: ##{{{(((...)))}}}##. One Group can contain another Group and there is no limit of imbrication.
```

Comment:
This is not fully correct. DIV only means something when you're outputting to HTML. This documentation should be generic. Or you should say that it's for HTML if you really think it's needed (as in "...another document (or a DIV tag when generating HTML)...").